src/uct/tcp/sockcm/sockcm_iface.h

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2017.  ALL RIGHTS RESERVED.
+ * Copyright (C) NVIDIA Corporation. 2019.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCT_SOCKCM_IFACE_H
+#define UCT_SOCKCM_IFACE_H
+
+#include "sockcm_def.h"
+#include "sockcm_md.h"
+
+#define UCT_SOCKCM_MAX_CONN_PRIV (UCT_SOCKCM_UDP_PRIV_DATA_LEN)
+
+typedef struct uct_sockcm_iface_config {
+    uct_iface_config_t       super;
+    unsigned                 backlog;
+    unsigned                 sock_id_quota;
```

Comment:
why do we need this? seems it's copied from rdmacm, but it's not really relevant for tcp?